CRM/Activity/BAO/Activity.php

```diff
@@ -886,6 +894,219 @@ public static function activityComponents() {
   }
 
   /**
+   * Get the activity Count.
+   *
+   * @param array $input
+   *   Array of parameters.
+   *    Keys include
+   *    - contact_id  int            contact_id whose activities we want to retrieve
+   *    - admin       boolean        if contact is admin
+   *    - caseId      int            case ID
+   *    - context     string         page on which selector is build
+   *    - activity_type_id int|string the activity types we want to restrict by
+   *
+   * @return int
+   *   count of activities
+   */
+  public static function &getActivitiesCount($input) {
+    $input['count'] = TRUE;
+    list($sqlClause, $params) = self::getActivitySQLClause($input);
+
+    //filter case activities - CRM-5761
+    $components = self::activityComponents();
+    if (!in_array('CiviCase', $components)) {
```

Comment:
dunno - but this is the code that we are going to remove again in master & it is what used to be in play - so that's not the issue